dask_expr/io/parquet.py

```diff
@@ -681,6 +686,20 @@ def _update_length_statistics(self):
 #
 
 
+def _set_engine(engine=None, meta=None):
```

Comment:
Can we rename to ``_set_parquet_engine``? This confused me a bit in ``_collections.py``